src/common/components/userMatchModal/userMatchModal.component.spec.js

```diff
@@ -25,7 +25,7 @@ describe( 'userMatchModal', function () {
 
   describe( '$onInit', () => {
     beforeEach( () => {
-      spyOn( $ctrl.verificationService, 'getContacts' ).and.returnValue( Observable.of( [] ) );
+      spyOn( $ctrl.verificationService, 'getContacts' ).and.callFake( () => Observable.of( [] ) );
```

Comment:
I've used returnValue when returning a primitive or immutable object. returnValue always returns the same object on multiple calls, meaning if the spy was called twice, return values would be equal(===). callFake will execute the method each time, meaning it will return different objects each time or allow return value to react to parameters passed to the spy.

In this case, I think they are equivalent.